tests/unit/test_rescue.py

```diff
@@ -83,36 +83,36 @@ def test_created_at_date_exists(rescue_sop_fx):
     """
     Verifies rescue.created_at datetime is set, and in the past.
     """
-    expected_time_differential = (datetime.now(tz=tzutc()) - rescue_sop_fx.created_at)
+    expected_time_differential = (pendulum.now() - rescue_sop_fx.created_at)
     assert expected_time_differential != 0
 
 
 def test_updated_at_date_exists(rescue_sop_fx):
     """
     Verifies rescue.updated_at is correct
     """
-    rescue_sop_fx._updatedAt = datetime(1990, 1, 1, 1, 1, 1, tzinfo=tzutc())
+    rescue_sop_fx._updatedAt = pendulum.datetime(1990, 1, 1, 1, 1, 1, tz=pendulum.tz.UTC)
 
     with rescue_sop_fx.change():
         rescue_sop_fx.system = 'UpdatedSystem'
 
-    assert rescue_sop_fx.updated_at != datetime(1990, 1, 1, 1, 1, 1, tzinfo=tzutc())
+    assert rescue_sop_fx.updated_at != pendulum.datetime(1990, 1, 1, 1, 1, 1, tz=pendulum.tz.UTC)
 
 
 def test_updated_at_raises_typeerror(rescue_sop_fx):
     """
     Verify Rescue.updated_at raises TypeError if given incorrect value,
     or is set to a date in the past.
     """
-    rescue_sop_fx._createdAt = datetime(1991, 1, 1, 1, 1, 1, )
+    rescue_sop_fx._createdAt = pendulum.datetime(1991, 1, 1, 1, 1, 1, )
```

Comment:
[style] We should be consistent, do we, or do we not append `tz=pendulum.tz.UTC`?